docs/events.rst

```diff
@@ -74,7 +78,9 @@ It's also possible to dump the raw events using the API.
        }
     ]
 
-The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release.
+The ``set`` parameter accepts an optional ``union`` or ``intersection`` argument to determine the behavior for filtering sets of tags (i.e. inclusive or exclusive). By default, Graphite uses a "lazy union" that will return any matching events for a given tag in a list of tags. This behavior is not intuitive and will therefore be deprecated in a future release. When matching using multiple tags, separate the tags with a space.
+
+*Note*: Prior to 0.10.0, the value of ``tags`` will be a string, with multiple tags being separated by a space.
```

Comment:
good point. In my case, I was talking about the output of get_data changing. I agree it's confusing so added what you suggested and clarified what I wrote.